src/lib/qboard.ts

```diff
@@ -8,6 +8,7 @@ import ClipboardHandler from "./clipboard";
 import StyleHandler, { Dash, Fill, Stroke, Style } from "./styles";
 import ActionHandler from "./action";
 import KeyboardHandler, { KeyMap } from "./keyboard";
+import FileHandler from "./files";
```

Comment:
can we move this before historyhandler? imports are ordered according to how theyre called in constructor xpxpxpxp